pghoard/pghoard.py

```diff
@@ -457,6 +458,7 @@ def main(args=None):
         description="postgresql automatic backup daemon")
     parser.add_argument("--version", action="version", help="show program version",
                         version=version.__version__)
+    parser.add_argument("-s", "--short-log", help="use non-verbose logging format", action="store_true")
```

Comment:
the opposite of verbose is usually `terse` . consider allow setting this in pghoard config?  there's also a ticket https://github.com/ohmu/pghoard/issues/90 about extensible logging configuration that's slightly related.